sci-libs/vtk/vtk-9.0.1.ebuild

```diff
@@ -139,6 +139,28 @@ DOCS=( CONTRIBUTING.md README.md )
 
 CHECKREQS_DISK_BUILD="3G"
 
+MY_CUDA_ARCH=
+
+get_cuda_arch() {
+	if [[ -z ${CUDA_ARCH} ]]; then
```

Comment:
One last thing, if we ever implement this in the eclass, there's no telling how we'll be using CUDA_ARCH variable exactly and it may conflict with vtk's implementation.

So VTK_CUDA_ARCH may make more sense until we have a real centralized implementation.